nodes.lua

```diff
@@ -16,6 +16,10 @@ minetest.register_node(":default:cloud", {
 	drop = "",
 	groups = {unbreakable = 1, not_in_creative_inventory = maptools.creative},
 	sounds = default.node_sound_defaults(),
+	on_drop = function(itemstack, dropper, pos)
+		maptools.msg(dropper)
+		return
```

Comment:
Superfluous `return`.

Reply:
okay I'll ditch the returns as they are superfluous